app/controllers/planning_applications/notes_controller.rb

```diff
@@ -30,20 +30,6 @@ def create
 
     private
 
-    def set_planning_application
-      planning_application = planning_applications_scope.find(planning_application_id)
-
-      @planning_application = PlanningApplicationPresenter.new(view_context, planning_application)
-    end
-
-    def planning_applications_scope
-      current_local_authority.planning_applications.includes(:notes)
```

Comment:
Did we want to do similar to how you did with recommendations for the planning application scope i.e. `super.includes(:notes)`

Reply:
I think there's a few other examples of this too (unless you deemed that the includes wasn't necessary)